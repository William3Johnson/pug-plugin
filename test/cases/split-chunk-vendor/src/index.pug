html
  head
    link(href=require('./styles.scss') rel='stylesheet')
    script(src=require('@test-fixtures/lorem') defer)
  body
    h1 Hello Pug!
    script(src=require('./main'))