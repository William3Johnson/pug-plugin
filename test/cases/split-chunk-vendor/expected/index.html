<!DOCTYPE html><html><head><script src="/assets/js/runtime.99c4f08c.js" defer></script><script src="/assets/js/vendors.74ff8d86.js" defer></script><script src="/assets/js/bootstrap.bundle.min.e255c241.js" defer></script></head><body><h1>Hello Pug!</h1></body></html>